HISTORY.rst

```diff
@@ -11,6 +11,9 @@ Release History
 
 - Flaky will no longer raise a KeyError for failed flaky tests.
 
+- Flaky will no longer raise a UnicodeEncodeError for flaky tests which raise exceptions
+  with non-ascii characters.
+
```

Comment:
This needs to be in a different section, called upcoming. It's not going to be part of the 2.1.1 release, since that's already out.

Side note, I need to update the CONTRIBUTING notes to reflect how to do this.